service_storage_admin/src/main/scala/no/uio/musit/microservice/storageAdmin/resource/StorageUnitResource.scala

```diff
@@ -58,13 +58,13 @@ class StorageUnitResource extends Controller {
   def listAll = Action.async {
     StorageUnitService.all.flatMap(list => {
       Future.sequence(list.map(unit => {
-        unit.`type` match {
-          case StorageType.StorageUnit =>
+        unit.storageType match {
```

Comment:
Pattern match on the Storage ADT instead of checking the String value of a property on.